### ISSUE TITLE
Error when coverting Peloton's New  "Just Work Out" workout type

### ISSUE THREAD

**Issue by xxsntxx** [philosowaffle/peloton-to-garmin#330]
**Describe the bug**
P2G is throwing an error when attempting to convert a "Just Work Out" Cardio class type. (Peloton expanded their "Just Work Out" class type options)

**To Reproduce**
1. In Peloton, log a "Just Work Out" cardio class type. 
2. Open P2G to sync "Just Work Out" cardio class type.
3. In P2G, error will occur when it attempts to download and convert workout. 

**Expected behavior**
P2G should pull logged "Just Work Out" Cardio from Peloton and convert to Garmin formart and upload to Garmin. 

**Logs**
```
Peloton To Garmin
[10:46:46 INF] Encoded FIT file C:\Users\SC\Downloads\Santi\P2G\working\00d17e0f48214718a2b1df663c501e5d_20_min_Hike_with_Olivia_Amato.fit
[10:46:46 INF] Backed up file ./output\fit\00d17e0f48214718a2b1df663c501e5d_20_min_Hike_with_Olivia_Amato.fit
[10:46:46 ERR] Failed to convert workout data 481d4bf677b34ca4925ee59d647ab12d_20_min_Just_Cardio_with_JUST_CARDIO ./working\downloaded\481d4bf677b34ca4925ee59d647ab12d_20_min_Just_Cardio_with_JUST_CARDIO.json
System.NullReferenceException: Object reference not set to an instance of an object.
   at Conversion.Converter`1.GetDistanceUnit(String unit) in D:\a\peloton-to-garmin\peloton-to-garmin\src\Conversion\IConverter.cs:line 515
   at Conversion.FitConverter.GetLapsBasedOnDistance(WorkoutSamples workoutSamples, DateTime startTime, Sport sport, SubSport subSport) in D:\a\peloton-to-garmin\peloton-to-garmin\src\Conversion\FitConverter.cs:line 653
   at Conversion.FitConverter.GetLaps(PreferredLapType preferredLapType, WorkoutSamples workoutSamples, DateTime startTime, Sport sport, SubSport subSport) in D:\a\peloton-to-garmin\peloton-to-garmin\src\Conversion\FitConverter.cs:line 589
   at Conversion.FitConverter.Convert(Workout workout, WorkoutSamples workoutSamples) in D:\a\peloton-to-garmin\peloton-to-garmin\src\Conversion\FitConverter.cs:line 49
   at Conversion.Converter`1.Convert(String format) in D:\a\peloton-to-garmin\peloton-to-garmin\src\Conversion\IConverter.cs:line 91
[10:46:53 INF] Uploaded workout ./working\upload\00d17e0f48214718a2b1df663c501e5d_20_min_Hike_with_Olivia_Amato.fit
[10:46:53 INF] Rate limiting, upload will continue after 4 seconds...
[10:46:58 INF] Sleeping for 43200 seconds...
```

**Installation (please complete the following information):**
 - OS: windows
 - Method: exe
 - Version: 2.0.1

**Additional context**
Peloton expanded their "Just Work Out" class type to allow just about any of the class type that are led by instructors. I am guessing that these new "Just Work Out" class types save a bit differently and P2G is not able to identify/convert correctly.

**Comment by philosowaffle** [comment]
Tentatively, I think this might be the same issue as #209, which was fixed in version [2.2.0](https://github.com/philosowaffle/peloton-to-garmin/releases/tag/vv2.2.0).

Can you test out the latest version [v3.3.0](https://github.com/philosowaffle/peloton-to-garmin/releases) and see if the error still shows up?

**Comment by philosowaffle** [comment]
Closing as stale.
